app/src/main/java/com/vero/tasky/agenda/data/repository/EventRepositoryImpl.kt

```diff
@@ -70,53 +55,48 @@ class EventRepositoryImpl(
                 val photoJobs = localPhotos.map {
                     launch { eventDao.insertLocalPhotoEntity(it.toLocalPhotoEntity(event.id)) }
                 }
-                (attendeeJobs + photoJobs).joinAll()
+                val deletedPhotoJobs = deletedPhotoKeys.map { key ->
+                    launch {
+                        eventDao.insertDeletedPhotoEntity(
+                            DeletedPhotoEntity(
+                                eventId = event.id,
+                                key = key
+                            )
+                        )
+                    }
+                }
+                (attendeeJobs + photoJobs + deletedPhotoJobs).joinAll()
             }
         }
     }
 
     override fun getEvent(id: String): Flow<AgendaItem.Event> {
-        return eventDao.loadEvent(id).map { it.toEvent() }
+        return eventDao.loadEventFlow(id).map { it.toEvent() }
     }
 
 
     override suspend fun updateEvent(
         event: AgendaItem.Event,
         deletedPhotoKeys: List<String>,
         isGoing: Boolean
-    ) : Result<AgendaItemUploadResult>{
+    ): Result<AgendaItemUploadResult> {
         val localPhotos = event.photos.filterIsInstance<AgendaPhoto.LocalPhoto>()
 
-        saveEvent(event = event, localPhotos = localPhotos)
+        saveEvent(event = event, localPhotos = localPhotos, deletedPhotoKeys = deletedPhotoKeys)
 
-        var skippedPhoto = 0
+        val multipartPhotos = multipartParser.getMultipartPhotos(localPhotos)
+        val skippedPhoto = localPhotos.size - multipartPhotos.size
```

Comment:
This is a number that counts how many photos were skipped, right?

So I would make it plural

skippedPhotos

Reply:
Also. Is it being used?
Im not seeing it

Reply:
I return it in a Result.success()